content/account_management/team.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Team
+kind: documentation
+autotocdepth: 2
+customnav: accountmanagementnav
+---
+
+## Add New Members
+
+1. To add members to a team, start by visiting the [Team Page](https://app.datadoghq.com/account/team).
+2. Enter the email address of the person you want to add to your team. Click **Invite Users**
```

Comment:
Enter the email address of the user you wish to invite to your Datadog account.

Reply:
👍